data/Briggs_2010/metadata.yml

```diff
@@ -15,8 +15,8 @@ contributors:
   data_collectors:
   - last_name: Briggs
     given_name: Amber
-    ORCID: .na.character
-    affiliation: unknown
+    ORCID: 0000-0001-5933-1064
+    affiliation: La Trobe University, Australia
```

Comment:
Can you remove her affiliation? She is someone who has "vanished" . I think the ORCID is fine.